aries_cloudagent/resolver/tests/test_did_resolver.py

```diff
@@ -82,7 +82,8 @@ async def setup(self, context):
     async def _resolve(self, profile, did, accept):
         if isinstance(self.resolved, Exception):
             raise self.resolved
-        return self.resolved.serialize()
+        meth = getattr(self.resolved, "serialize", None)
+        return meth() if callable(meth) else self.resolved
```

Comment:
I think it would be better to instead check types:
```suggestion
        if isinstance(self.resolved, dict:
            return self.resolved
        return self.resolved.serialize()
```

Reply:
I agree. I will update the code.